src/app/(main)/(user)/borrow-form/page.tsx

```diff
@@ -115,207 +148,133 @@ const LMBInventoryForm = () => {
-                          required
-                          className="w-full"
+                        <TimePicker
+                          label="Time Borrowed"
+                          value={field.value}
+                          onChange={(newTime) => field.onChange(newTime)}
                         />
                       </FormControl>
                       <FormMessage />
                     </FormItem>
                   )}
                 />
 
-              <div className="grid grid-cols-2 gap-4 mb-4">
-
+                {/* Date Returned */}
                 <FormField
-                  name="quantity"
+                  name="dateReturned"
                   render={({ field }) => (
                     <FormItem>
-                      <FormLabel>Quantity</FormLabel>
+                      <FormLabel>Date Returned</FormLabel>
                       <FormControl>
-                        <Input
-                          type="number"
-                          placeholder="Quantity"
-                          {...field}
-                          required
-                          className="w-full"
-                        />
-                      </FormControl>
-                      <FormMessage />
-                    </FormItem>
-                  )}
-                />
-                <FormField
-                  name="unit"
-                  render={({ field }) => (
-                    <FormItem>
-                      <FormLabel>Unit</FormLabel>
-                      <FormControl>
-                        <Input
-                          placeholder="Unit (e.g., liters, grams)"
-                          {...field}
-                          required
-                          className="w-full"
+                        <DatePickerWithPresets
+                          date={field.value}
+                          setDate={(newDate) => field.onChange(newDate)}
                         />
                       </FormControl>
                       <FormMessage />
                     </FormItem>
                   )}
                 />
 
+                {/* Time Returned */}
                 <FormField
-                  name="borrowDetails"
+                  name="timeReturned"
                   render={({ field }) => (
                     <FormItem>
-                      <FormLabel>Borrower Details</FormLabel>
+                      <FormLabel>Time Returned</FormLabel>
                       <FormControl>
-                        <Input
-                          placeholder="Details of Borrower"
-                          {...field}
-                          required
-                          className="w-full"
+                        <TimePicker
+                          label="Time Returned"
+                          value={field.value}
+                          onChange={(newTime) => field.onChange(newTime)}
                         />
                       </FormControl>
```

Comment:
Type '{ label: string; value: any; onChange: (newTime: any) => void; }' is not assignable to type 'IntrinsicAttributes & TimePickerProps'.
  Property 'label' does not exist on type 'IntrinsicAttributes & TimePickerProps'.ts(232)
  
  Parameter 'newTime' implicitly has an 'any' type.ts(7006)
(parameter) newTime: any

Reply:
Add eslint extension to see these errors. Check setup docs for frontend.